airbyte-integrations/connectors/source-mssql/src/main/kotlin/io/airbyte/integrations/source/mssql/config_spec/MsSqlServerSourceConfigurationSpecification.kt

```diff
@@ -0,0 +1,122 @@
+/* Copyright (c) 2024 Airbyte, Inc., all rights reserved. */
+package io.airbyte.integrations.source.mysql
+
+import com.fasterxml.jackson.annotation.JsonGetter
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.annotation.JsonPropertyDescription
+import com.fasterxml.jackson.annotation.JsonPropertyOrder
+import com.fasterxml.jackson.annotation.JsonSetter
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaDefault
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaInject
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings
+import io.airbyte.cdk.command.CONNECTOR_CONFIG_PREFIX
+import io.airbyte.cdk.command.ConfigurationSpecification
+import io.airbyte.integrations.source.mssql.config_spec.MicronautPropertiesFriendlyMsSqlServerEncryption
+import io.airbyte.integrations.source.mssql.config_spec.MicronautPropertiesFriendlyMsSqlServerReplicationMethodConfiguration
+import io.airbyte.integrations.source.mssql.config_spec.MsSqlServerEncryptionConfiguration
+import io.airbyte.integrations.source.mssql.config_spec.MsSqlServerReplicationMethodConfiguration
+import io.micronaut.context.annotation.ConfigurationBuilder
+import io.micronaut.context.annotation.ConfigurationProperties
+import jakarta.inject.Singleton
+
+/**
+ * The object which is mapped to the Mysql source configuration JSON.
```

Comment:
please do a string search&replace for `/mysql/mssql/` in this source tree before we forget to do so :)